src/lib/operators/jit_operator_wrapper.cpp

```diff
@@ -0,0 +1,86 @@
+#include "jit_operator_wrapper.hpp"
+
+namespace opossum {
+
+JitOperatorWrapper::JitOperatorWrapper(const std::shared_ptr<const AbstractOperator> left, const bool use_jit,
```

Comment:
Naming: This is the wrapper for JIT operators - how does `bool use_jit` make sense?

Reply:
The boolean specifies, whether the pipeline should be compiled or interpreted. How about an enum `ExecutionMode`?

Reply:
Sounds good!